.github/workflows/iroha2.yml

```
@@ -6,49 +6,35 @@ on:
 
```

If we synchronize tags and branches and unify the `iroha2` and `iroha2-stable` __branches__ as I mentioned in https://github.com/hyperledger/iroha/pull/2576#discussion_r937453231,

```suggestion
on:
  push:
    branches: [iroha2-stable]
```

Also, the workflow filenames will be renamed to `iroha2-stable.yml`, etc.